analyses/malawi/scripts/mwi_chirps_dry_spell_detection.R

```diff
@@ -728,10 +729,61 @@ data_long %>%
   ungroup() %>%
   summary(n_dry_days)
 
+############
+# LOOK AT FALSE ALARMS
+############
+#can comment out all code above and read the saved file
+# data_long_mean_values <- readRDS(paste0(dry_spell_processed_path, "data_long_mean_values.RDS"))
+
+# distribution of rolling sums in southern region for Jan and Feb
+data_long_mean_values %>%
+  mutate(southern_region = ifelse(startsWith(pcode, 'MW3') == TRUE, 1, 0)) %>% # identify southern region records
+  filter(southern_region == 1 & month %in% c(1,2)) %>%
+  summary()
+
+# compute max/min 10-day rolling sum per month for southern region
+fa <- data_long_mean_values %>%
+        mutate(southern_region = ifelse(startsWith(pcode, 'MW3') == TRUE, 1, 0)) %>% # identify southern region records
+        filter(southern_region == 1) %>% # select records for southern region
+        group_by(year, month) %>%
+        summarise(max_10d_sum = max(round(rollsum_10d, 0), na.rm = T),
+                  min_10d_sum = min(round(rollsum_10d,0), na.rm = T))
+
+# was there a day within a 10d period with a rolling sum <= 10?
+fa$with_10d_10mm <- ifelse(fa$min_10d_sum <= 10, 1, 0)
+
+write.csv(fa, file = paste0(dry_spell_processed_path, "false_alarms_deep_dive.csv"), row.names = FALSE)
```

Comment:
Could we change this to a more meaningful name, e.g. `mwi_chirps_10days_le10mm.csv`

Reply:
@joseepoirier I have felt so free to change this, let me know if you had prefer not to.

Reply:
yes, no problem.